bcauth/jinja2/oauth2_provider/application_confirm_delete.html

```diff
@@ -0,0 +1,18 @@
+{% extends "account/profile.html" %}
+
+{% block head_subtitle %} - {{ _("OAuth2 Client Application") }} - {{ _("Delete") }}{% endblock %}
+{% block body_title %}{{ _("OAuth2 Client Application") }} - {{ _("Delete") }}{% endblock %}
+
+{% block content %}
+    <h3>{{ _("Are you sure to delete the application %(app_name)s?") | format(app_name=application.name) }}</h3>
```

Comment:
``_`` (underscore) is actually Django's ``ugettext``, which doesn't support new style ``gettext`` (it errors if you try to use it).  Another thing to figure out when we actually localize the API...

https://github.com/mdn/browsercompat/blob/master/wpcsite/jinja2.py#L28-L30